x-pack/plugins/security/server/routes/role_mapping/feature_check.ts

```diff
@@ -49,7 +49,7 @@ export function defineRoleMappingFeatureCheckRoute({ router, logger }: RouteDefi
       const esClient = (await context.core).elasticsearch.client;
       const { has_all_requested: canManageRoleMappings } =
         await esClient.asCurrentUser.security.hasPrivileges({
-          body: { cluster: ['manage_security'] },
+          body: { cluster: ['read_security'] },
```

Comment:
Role mappings already support read only view https://github.com/elastic/kibana/blob/main/x-pack/plugins/security/public/management/role_mappings/role_mappings_management_app.tsx#L88

Reply:
Long term, would it make more sense to change this internal endpoint to '/internal/security/_check_security_features'? We're really using this to check which features are enabled, with a minimum requirement of "security_read" priv.

Do we know if we have any external (non-Kibana) callers to this API? If not, is it ok to move/change the API now?

cc: @legrego @azasypkin

Reply:
> Long term, would it make more sense to change this internal endpoint to '/internal/security/_check_security_features'? In that case we would rename 'canManageRoleMappings' to 'canReadSecurity'. We're really using this (all use cases that I can find) to check which features are enabled, with a minimum requirement of "security_read" priv.

++ we shouldn't be calling a Role Mappings internal API from the Role Management screen. This isn't a new problem though, it exists in `main` since the introduction of remote index privileges. I'll leave the "when" up to y'all: I don't have a strong preference if we refactor in this PR, or create a follow-up issue to address as time permits.

> Do we know if we have any external (non-Kibana) callers to this API? If not, is it ok to move/change the API now?

We shouldn't have external callers to this API. We reserve the right to change `/internal` APIs as needed. If we are aware that an `/internal` API has become defacto public, then it changes the equation a bit, but I think we are safe in this instance.